libs/feature/catalog/src/lib/organisations/service/organisations-from-metadata.service.spec.ts

```diff
@@ -192,21 +219,21 @@ describe('OrganisationsFromMetadataService', () => {
           {
             description: null,
             emails: ['rolf.giezendanner@are.admin.ch', 'john.doe@are.admin.ch'],
-            logoUrl: null,
+            logoUrl: undefined,
```

Comment:
I don't know if there was a purpose to be `null` instead of `undefined` ?

Reply:
I don't think. We just check nullity/emptiness for hydrating.